lib/checking.rb

```diff
@@ -0,0 +1,42 @@
+require './lib/account'
+
+module Bank
+  class CheckingAccount < Account
+    MAX_CHECKS = 3
+    FEE = 100
+    OVERDRAFT = -1000
+    def initialize(id, balance, open_date = "today", owner = nil)
+      super(id, balance, open_date, owner)
+      @type = "Checking"
+      @fee = 100
+      @num_checks = 0
+    end
+
+    def withdraw_using_check(amount)
+      if amount < 0
+        puts "You cannot withdraw a negative amount of money."
+      else
+        puts "---CHECK WITHDRAWAL---".colorize(:blue)
+        # This is a shorthand if-else statement
+        @num_checks > 2 ? (penalty = 200) : (penalty = 0)
```

Comment:
Nice use of the single-line conditional here. You could also re-write this as:
`penalty = (@num_checks > 2) ? 200 : 0`